src/Controller/CourseListController.php

```diff
@@ -84,15 +84,18 @@ public function build() {
     $build['#attached']['library'][] = 'anu_lms/application';
     $build['#attached']['library'][] = 'core/drupalSettings';
     $build['#attached']['drupalSettings']['anu_settings'] = $this->anulmsSettings->getSettings();
-    $build['#attached']['drupalSettings']['anu_courses'] = $normalizedCourses;
+    $build['#attached']['drupalSettings']['anu_courses'] = [
+      'courses' => $normalizedCourses,
+      'page_title' => t('Courses'),
+      'all_courses_label' => t('All courses'),
```

Comment:
It's not completely clear from the variable name what this variable for
Can we make the var name more specific?

Reply:
Updated. Thanks.